clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1176,10 +1176,11 @@ public static String toHtml() {
         for (ApiKeys key : ApiKeys.values()) {
             // Key
             b.append("<h5>");
+            b.append("<a name=\"The_Messages_" + key.name + "\">");
```

Comment:
No need, if it's valid then the `name` version is fine. I'm probably just out of date.